topics/task-worker.yaml

```diff
@@ -0,0 +1,25 @@
+pipeline: taskworker
+description: |
+  Taskworker tasks to be executed
+services:
+  producers:
+    - getsentry/sentry
+  consumers:
+    - getsentry/sentry
+schemas:
+  - version: 1
+    compatibility_mode: none
+    # TODO(mark) protobuf support
+    type: json
+    resource: taskworker.v1.schema.json
```

Comment:
Ideally this would be `type: protobuf` and we would have a way to link to the protobuf IDL/types.